### PR TITLE
Add Kokkos profiling labels

### PR DESCRIPTION
**Pull request by streeve**
Update to grid operators that is NOT backwards compatible

### CODE REVIEW

File: src/Picasso_ParticleInit.hpp [hunk 1]
```diff
@@ -47,6 +47,7 @@ void filterEmpties( const ExecutionSpace& exec_space,
         Kokkos::ViewAllocateWithoutInitializing( "empties" ),
         std::min( num_particles - local_num_create, local_num_create ) );
     Kokkos::parallel_scan(
+        "find_empty_particles",
```
Comment (dalg24) [inline src/Picasso_ParticleInit.hpp:50]: Consider prefixing with something like `Picasso::`

Comment (sslattery) [reply src/Picasso_ParticleInit.hpp:50]: I agree with this

**Review by sslattery — CHANGES_REQUESTED** [review]
I think this is fine if we add the prefix per @dalg24 suggestion

File: src/Picasso_ParticleInit.hpp (hunk 1 above)
Comment (streeve) [reply src/Picasso_ParticleInit.hpp]: Fixed

**Comment by streeve** [comment]
@sslattery ready for re-review - I added more labels as well

**Pull request merged**